### DISCUSSION TITLE
Typescript Error for Spotlight.

### DISCUSSION THREAD

**Discussion by lingayya0124** [mantinedev/mantine#1959] [Q&A] [answered]
I am getting a typescript warning when using Spotlight on the latest version of mantine 5.0.0
here is screenshot
![Screenshot from 2022-08-02 12-19-54](https://user-images.githubusercontent.com/69788802/182311869-e8b00196-0c63-45e1-a0b6-de8d59653571.png)

**Comment by zhulien-ivanov** [comment]
The error is self-explanatory. The event that gets passed to `openSpotlight` is not compatible with your input param type. What input are you expecting in `openSpotlight`? If it is not the native event then use:
`<UnstyledButton onClick={() => openSpotlight()}>`.

Show the code of your `openSpotlight` function.

**Comment by rtivital** [comment, marked as answer]
Should be `<UnstyledButton onClick={() => openSpotlight()} />`

**Comment by lingayya0124** [comment]
Thank you @rtivital @zhulien-ivanov now the typescript warning gone.
